apiary.apib

```diff
@@ -929,6 +929,22 @@ Update a company.
         + language: `en` (string, optional, nullable)
         + responsible_user_id: `0ea94804-401d-4dbd-a577-c2d60998f798` (string, optional, nullable)
         + remarks: `Met at expo` (string, optional, nullable)
+        + custom_fields (array, optional)
+            + (object)
+                + id: `bf6765de-56eb-40ec-ad14-9096c5dc5fe1` (string)
+                + One Of
+                    + value: `092980616` (string)
+                    + value: `123` (number) - For integer, number, money and auto-increment fields
+                    + value: `foo`,`bar` (array[string]) - For multiple selection fields
+                    + value: true (boolean) - For Yes/No fields
+                    + value (object) - For related Teamleader objects
+                        + type: `company` (enum[string])
+                            + Members
+                                + company
+                                + contact
+                                + product
+                                + user
+                        + id: `bf6765de-56eb-40ec-ad14-9096c5dc5fe1` (string)
```

Comment:
Q: since this is a copy paste, wouldn't it be better to add a `CustomFieldValue` datastructure?

Reply:
Done!

 :ok: by @dietrichm on 04/05/2018 13:14:30